lib/pmf_nfpse_generator.rb

```diff
@@ -219,15 +217,32 @@ def billing_date_cannot_be_in_the_future
     date_current = Date.current
     date = billing_date.try(:to_date)
     if date != date_current && date > date_current
-      errors.add(:billing_date, I18n.t("billing_date.cannot_be_in_the_future"))
```

Comment:
@silviofernandesrd i remember about that, in RDContas we use this mgs to show to user whats going on, take care.